RHEL6_7/networking/dovecot/check

```diff
@@ -53,7 +62,7 @@ then
     rm -f $TMPF1 $TMPF2
   fi
 else
-  log_error "Cannot use doveconf to parse the configuration files."
+  log_high_risk "Dovecot configuration is invalid. Fix it or remove the dovecot package before proceeding with a upgrade"
```

Comment:
"The Dovecot configuration is invalid. Fix it or remove the dovecot package before proceeding with the upgrade"